src/app/project/vr-coaster/vrCoaster.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/jsx-key */
+import Underline from "@/components/Content/Underline/Underline";
+import { ProjectData } from "@/types/projectInterface";
+import { CATEGORY } from "@/types/types";
+import { convertScheduleToReactNode } from "@/utils/converter";
+import { createDate } from "@/utils/date";
+
+// TODO: タグ確認
+// TODO:企画紹介文の!!確認
+
+const schedule = {
+  day1: {
+    startDate: createDate({
+      date: 1,
+      hour: 10,
+      minute: 0,
+    }),
+    endDate: createDate({
+      date: 1,
+      hour: 16,
+      minute: 0,
+    }),
+  },
+  day2: {
+    startDate: createDate({
+      date: 2,
+      hour: 10,
+      minute: 0,
+    }),
+    endDate: createDate({
+      date: 2,
+      hour: 15,
+      minute: 30,
+    }),
+  },
+};
+
+const place = "23号館2321講義室";
+
+// 変数名を変更(ex. miniGame)
+// ファイル名も変更(ex. miniGame.tsx)
+export const vrCoaster: ProjectData = {
+  // ContentBoxに含まないデータ
+  link: "vr-coaster", // 複数単語の場合はハイフンつなぎで記述
+  name: "VRコースター",
+  tags: [],
+  category: [CATEGORY.STUDENT, CATEGORY.EXPERIENCE],
+  summary: "",
+
+  // ProjectBoxに変換する必要があるデータ
+  schedule,
+  place,
+
+  projectBoxList: [
+    {
+      title: "企画紹介",
+      content: (
+        <p>
+          迫力満点のVRコースターに乗ってみませんか⁈仮想現実の世界でスリル満点の体験を、リアルな動きと共に楽しめます！
+          手作りの木製カートに乗り込み、心臓が高鳴るジェットコースターの世界をお楽しみください‼
+        </p>
```

Comment:
```suggestion
        <p>
          迫力満点のVRコースターに乗ってみませんか⁈仮想現実の世界でスリル満点の体験を、リアルな動きと共に楽しめます！
          <br />
          手作りの木製カートに乗り込み、心臓が高鳴るジェットコースターの世界をお楽しみください‼
        </p>
```
IMO
若干読みにくいから改行ほしいです